public/data/dmitrii-kilishek.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "Dmitrii Kilishek",
+  "type": "personal",
+  "bio": "Hey dear visitor! I'm Dmitrii. A 🇺🇸 based Software Tester, Lifelong Learner, Former Sysadmin, In IT Since 1994, From: 🇷🇺 with ❤️",
```

Comment:
```suggestion
  "bio": "Hey dear visitor! I'm Dmitrii. A 🇺🇸 based Software Tester, Lifelong Learner, Former Sysadmin. I've been in IT since 1994, From: 🇷🇺 with ❤️",
```

Update bio.